lib/rotoscope.rb

```diff
@@ -34,17 +34,16 @@ def self.trace(dest, blacklist: [], flatten: false)
   end
 
   def flatten(dest)
-    io_given = false
-    dest_file = if dest.respond_to?(:to_io)
-      io_given = true
-      dest.to_io
+    io_given = if dest.respond_to?(:puts)
```

Comment:
Why do we support so many different ways of using the library?  Don't we have only a single use case for it rotoscope?